### ISSUE TITLE
accessibility: Loading DX7 patches

### ISSUE THREAD

**Issue by insomnimus** [asb2m10/dexed#515]
I was pleasantly surprised to see this plugin be screen-reader accessible! However I couldn't find a way to load DX7 patches. There are 2 unlabeled buttons next to the preset selector, but pressing them seems to do nothing. Am I missing something?
